x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/plain_column_renderer.tsx

```diff
@@ -40,9 +40,9 @@ export const plainColumnRenderer: ColumnRenderer = {
     linkValues?: string[] | null | undefined;
   }) =>
     values != null
-      ? values.map((value) => (
+      ? values.map((value, i) => (
           <FormattedFieldValue
-            key={`plain-column-renderer-formatted-field-value-${timelineId}-${columnName}-${eventId}-${field.id}-${value}`}
+            key={`plain-column-renderer-formatted-field-value-${timelineId}-${columnName}-${eventId}-${field.id}-${value}-${i}`}
```

Comment:
Just curious, were the `${eventId}-${field.id}-${value}` not unique? I ask because we ran into an issue on the resolver when we introduced indices into keys because reacts resolution on re-renders would be jumbled if the data changed

Reply:
it would throw the error on the nested field
<img width="390" alt="Screen Shot 2021-02-19 at 12 08 15 PM" src="https://user-images.githubusercontent.com/6935300/108550139-5ff81500-72ab-11eb-888e-06c2e8f7c4e3.png">

Reply:
i wonder if this was before or after i filtered for unique values though, ill take a second look